src/main/kotlin/uk/gov/dluhc/printapi/service/aed/AnonymousElectorDocumentSearchService.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.dluhc.printapi.service.aed
+
+import org.springframework.data.domain.Page
```

Comment:
Hmmm .... I'm not sure about the service class knowing (and exposing via the method return type) JPA / DB related stuff .... feels like the wrong coupling ?

What do we do in VCA in this respect?

Reply:
This will eventually be passed on from Controller when we start doing pagination properly. So everything from service will be removed/moved out to mapper etc. In VCA the endpoint query params supplies those values and passes them onto repository. (Also we use specification builder which I don't think we need in this case here and it's a simple search queries) .. I can add a TODO to say remove or move this to mapper when endpoint supports pagination etc.

Reply:
.. or I can pass in `SearchQueryStringParameters` in service which will be populated by Controller?